extensions/amp-analytics/0.1/events.js

```diff
@@ -457,6 +457,15 @@ export class AmpStoryEventTracker extends CustomEventTracker {
     const repeat =
       storySpec['repeat'] === undefined ? true : storySpec['repeat'];
     const pageDetails = vars['pageDetails'];
+    const tagName = config['tagName'];
+
+    if (
+      tagName &&
+      pageDetails['tagName'] &&
+      tagName.toLowerCase() !== pageDetails['tagName']
```

Comment:
Also `.toLowerCase()` the `pageDetails` one

Reply:
I am doing that on the `updateDetails` method in `story-analytics.js` when reading the element. Would you prefer it to be here instead?